src/app/components/Item/ItemFilters.jsx

```diff
@@ -30,10 +30,25 @@ const ItemFilters = (
   const [selectedYear, setSelectedYear] = useState(query.date || '');
   const [selectedFilters, setSelectedFilters] = useState(initialFilters);
   const [selectedFilterDisplayStr, setSelectedFilterDisplayStr] = useState('');
+  // normalize item aggregations by dropping values with no label and combining duplicate lables
+  const reducedItemsAggregations = JSON.parse(JSON.stringify(itemsAggregations))
+  reducedItemsAggregations.forEach((agg) => {
+    const values = agg.values
+    const reducedValues = {}
+    values.filter(value => value.label && value.label.length)
```

Comment:
Nitpick: use the shorthand `value => value.label?.length`.

Reply:
fixed